config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes-sigs/kubebuilder:
+
+    - name: post-kube-rbac-proxy-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-kubebuilder-kube-rbac-proxy, sig-k8s-infra-gcb
+      decorate: true
+      run_if_changed: 'TAG'
```

Comment:
@camilamacedo86 you probably want to remove this line. It states the job will just run in case a file called "TAG" is changed, which I guess it is not the case for this branch. IIUC, in this branch every time there is a new commit it means "create a new staging image" (that can be promoted after it)